lib/puppet/file_system/file_impl.rb

```diff
@@ -155,13 +155,19 @@ def chmod(mode, path)
     FileUtils.chmod(mode, path)
   end
 
+  def chown(uid, gid, path)
```

Comment:
As we don't have a `Puppet::FileSystem.chown` method, I'm thinking we should just inline the `FileUtils.chown` call below. Otherwise the posix file_impl is diverging from the "interface"